src/cards/ha-camera-card.html

```diff
@@ -44,7 +44,7 @@
     <div class='caption'>
       [[computeStateName(stateObj)]]
       <template is='dom-if' if='[[!imageLoaded]]'>
-        (Image not available)
+        ([[localize('card.camera.not_available')]])
```

Comment:
Since this is only tied to the interface, let's move it to `ui.card.camera.not_available`.